src/scmdata/ops.py

```diff
@@ -566,14 +620,52 @@ def integrate(self, out_var=None):
     out_var : str
         If provided, the variable column of the output is set equal to
         ``out_var``. Otherwise, the output variables are equal to the input
-        variables, prefixed with "Cumulative " .
+        variables, prefixed with "Cumulative ".
+
+    out_unit : str
```

Comment:
I'd leave this out too, users can just call `.convert_unit` after calling `integrate`. I think encouraging chaining of methods rather than every method having every argument makes more sense for long-term maintenance (separation of concerns being the main one I'm thinking about here).